core-plugins/docs/ExcelInputReader-batchsource.md

```diff
@@ -0,0 +1,128 @@
+# ExcelInputReader Batch Source
+
+
+Description
+-----------
+The Excel Input Reader provides user the ability to read data from one or more Excel file(s).
+
+The Input Reader supports following types of Excel file(s):
+Microsoft Excel 97(-2007) file format
+Microsoft Excel XML (2007+) file format
+
+
+Use Case
+--------
+The Excel Input Reader is used to read excel file(s) and converts the rows to structured records based
+on the column names, column-label mapping and column-type mapping provided by the user. Also keeps track
+of all the processed excel files in a memory table provided by the user. So, that if user has the option
+not to reprocess a particular file.
+
+
+Properties
+----------
+
+**filePath:** Path of the excel file(s) to be read. Supports below formats:
+
+      Microsoft Excel 97(-2007) file format
+      Microsoft Excel XML (2007+) file format
+
+**filePattern:** Regex pattern to select specific excel file(s) from the path provided
+in **filePath** input.
+
+**memoryTableName:** KeyValue table name to keep the track of processed files. This can be
+a new table or existing one.
+
+**reprocess:** Specify whether the files mentioned in the memory table should be reprocessed or not.
+
+**sheetName:** Sheet name in the excel files to be processed.
+
+**sheetNo:** Sheet number in the excel files to be processed.
+
+**columnList:** Specify the excel column names which needs to be extracted from the excel sheet.
+Column name has to be same as excel column name; for example: A, B, etc.
+
+**columnMapping:** List of the excel column names to be renamed. The key specifies the name of the
+excel column to be renamed, with its corresponding value specifying the new name for that column.
+Column name has to be same as excel column name; for example: A, B, etc.
+
+**skipFirstRow:** Specify whether the first row in the excel sheet needs to be processed or not.
+
+**terminateIfEmptyRow:** Specify whether processing needs to be terminated in case an empty row is
+encountered while processing excel files.
+
+**rowsLimit:** Maximum row limit for each sheet to be processed. If, the limit is not provided then
+all the rows in the sheet will be processed.
+
+**outputSchema:** Mapping of excel column names in the output schema to data types. Consists of
+a comma-separated list. This input is mandatory if no inputs for 'columnList' has been provided.
+Column name has to be same as excel column name; for example: A, B, etc.
+
+If type has not been provided for a column mentioned in **columnList** input, then output data type
+of that column will be **string**.
+
+**ifErrorRecord:** Specifies the action to be take in case of an error.
+
+**errorDatasetName:** Table name to keep the error record encountered while processing the excel file(s).
+
+
+Example
+-------
+
+This example reads data from a hdfs excel file "hdfs://10.222.73.37:9000/tmp/inventory.xlsx"  and parses it
```

Comment:
Also it seems like based on sentence that it can only read that file. So, I would recommend removing it completely.